plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMinioParquetConnectorSmokeTest.java

```diff
@@ -31,4 +32,16 @@ protected boolean isFileSorted(Location path, String sortColumnName)
     {
         return checkParquetFileSorting(fileSystem.newInputFile(path), sortColumnName);
     }
+
+    @Test
+    public void testQueryHadoopFsTableInNativeFsWithS3anScheme()
+    {
+        String bucket = "trino-ci-test";
+        String tableName = "sample_table1";
+        hiveMinioDataLake.getMinio().createBucket(bucket);
```

Comment:
We create here a bucket which we don't seem to remove after the test.
The name `trino-ci-test` is also the name used on AWS S3 and this can lead to potential confusions